api/models/process.go

```diff
@@ -217,6 +218,7 @@ func ListProcesses(app string) ([]*Process, error) {
 		}
 
 		if ec2i == nil {
+			// Error out if it didn't find an ec2 instance? Maybe just continue?
```

Comment:
For now, indeed it seems ok. I was thinking along the lines if one ECS/EC2 call fails, there could still be others that working fine. Do we want to ignore those?